### ISSUE TITLE
Add others key names for numpad "minus" and "plus" keys

### ISSUE THREAD

**Issue by tupo2** [mpv-player/mpv#11930]
### Expected behavior of the wanted feature
When binding numpad "minus" and "plus" keys, it also do it for the "minus" and "plus" keys to the left of  "backspace".
It would be handy to add others key names for numpad keys, in order to dissociate them.

Example:
```
KP_MINUS or KP_SUBTRACT
KP_PLUS or KP_ADD
```

**Comment by tupo2** [comment]
Since mpv 0.36.0 "KP_DEC" has been replaced with "."
Can we restore the old key name?

**Comment by butterw** [comment]
KP_DEC is still available for me in todays mpv v0.36.0-88-g598e3017 on Win10.
# List of special keys: mpv --input-keylist
# Keybindings testing mode: mpv --input-test --force-window --idle

Adding specific bindings for numpad +/- keys would be useful (especially for non-english keyboards).

**Comment by tupo2** [comment]
> KP_DEC is still available for me in todays mpv v0.36.0-88-g598e3017 on Win10. -List of special keys: mpv --input-keylist -Keybindings testing mode: mpv --input-test --force-window --idle
> 
> Adding specific bindings for numpad +/- keys would be useful.

You are right, it's not mpv, it's because I added a variant (oss) to my keyboard configuration file.
Thanks.

**Comment by tupo2** [comment]
Fixed in 0.39.0.

New key names:
```
KP_SUBTRACT
KP_ADD
KP_DIVIDE
KP_MULTIPLY
```
